### PR TITLE
Fix images

### PR DESCRIPTION
**Pull request by d2bit**
https://trello.com/c/hVv1NzZr/8-3-arreglar-los-fallos-de-subida-de-imagenes-en-las-news
Default User-Avatar image, Twitter logo & cuadradito_rojo was getting 404

### CODE REVIEW

**Comment by scruti** [comment]
:+1:

File: app/views/news_tematica/news_tematicas/_footer.html.haml [hunk 1]
```diff
@@ -2,13 +2,17 @@
...
   %tr
     %td.social
       %span.texto Síguenos en
       .iconos
-        = link_to_con_estadisticas image_tag('http://twitter.com/images/resources/twitter-bird-blue-on-white.png', alt: 'Twitter'), ::NewsTematica::Personalizacion.twitter_url, @news_tematica, class: 'icono'
-        = link_to_con_estadisticas image_tag('http://www.facebookbrand.com/img/assets/asset.f.logo.lg.png', alt: 'Facebook'), ::NewsTematica::Personalizacion.facebook_url, @news_tematica,  class: 'icono'
+        = link_to_con_estadisticas image_tag(image_url('iconos/twitter.png'), alt: 'Twitter'),
+          ::NewsTematica::Personalizacion.twitter_url, @news_tematica, class: 'icono'
+        = link_to_con_estadisticas image_tag(image_url('iconos/facebook.png'), alt: 'Facebook'),
+          ::NewsTematica::Personalizacion.facebook_url, @news_tematica,  class: 'icono'
```
Comment (Fernan2) [inline app/views/news_tematica/news_tematicas/_footer.html.haml]: The original resources were dropped?? what a fail, FB & Twitter!!

**Comment by Fernan2** [comment]
+1

File: app/views/news_tematica/news_tematicas/_footer.html.haml (hunk 1 above)
Comment (d2bit) [inline app/views/news_tematica/news_tematicas/_footer.html.haml]: only twitter image were missing, but I think is best if we serve both of them.

**Pull request merged**